scripts/ci/generate_pr_summary.py

```diff
@@ -47,7 +47,7 @@ def generate_pr_summary(github_token: str, github_repository: str, pr_number: in
         app_blob = viewer_bucket.blob(f"commit/{commit_short}/index.html")
         if app_blob.exists():
             print(f"Found web assets commit: {commit_short}")
-            found["hosted_app"] = f"https://app.rerun.io/commit/{commit_short}"
+            found["hosted_app"] = f"https://rerun.io/viewer/c/{commit_short}"
```

Comment:
`/c/` seems unnecessary short and cryptic imho.

For reference, `GitHub` uses `commit`:

`https://github.com/rerun-io/rerun/commit/c504eba`